scipy/stats/tests/test_distributions.py

```diff
@@ -3192,6 +3192,31 @@ def test_fit(self):
         assert_almost_equal(scale2, .02503, 4)
         assert_almost_equal(loc2, .03354, 4)
 
+    @pytest.mark.xfail(reason="Bug. Possibly fixed by #14476.")
+    @pytest.mark.parametrize(
+        "alpha,beta,delta,gamma",
+        [
+            (1.5, 0.4, 2, 3),
+            (1.0, 0.4, 2, 3),
+        ]
+    )
+    @pytest.mark.parametrize(
+        "parametrization", ["S0", "S1"]
+    )
+    def test_fit_rvs(self, alpha, beta, delta, gamma, parametrization):
+        """Test that fit agrees with rvs for each parametrization."""
+        stats.levy_stable.parametrization = parametrization
+        data = stats.levy_stable.rvs(
+            alpha, beta, loc=delta, scale=gamma, size=10000, random_state=1234
+        )
+        fit = stats.levy_stable._fitstart(data)
+        alpha_obs, beta_obs, delta_obs, gamma_obs = fit
+        assert_allclose(
+            [alpha, beta, delta, gamma],
+            [alpha_obs, beta_obs, delta_obs, gamma_obs],
+            rtol=0.01,
+        )
```

Comment:
Interesting, the alpha and beta estimates here are pretty accurate, but the delta and gamma ones are not. You're right this is maybe related to https://github.com/scipy/scipy/pull/14476 or the usual parameterization confusion of the field.

For a future PR, reading through McCulloch's paper again and doing some extra testing could probably resolve it. In any case, I think this is only used to bootstrap the fit's parameter optimization afterwards, so it's probably not a huge issue regardless.

Reply:
I see. It probably makes more sense to test fit directly. That can be left for later though.

Reply:
The functionality of _fitstart() is useful (unofficially) to anyone who has quantiles of a distribution, but not raw samples, and wants to estimate a stable distribution from the quantiles.  That's what I was trying to do that prompted [scipy#14476](https://github.com/scipy/scipy/pull/14476).  But of course that's outside the public API for stable distributions.